Wikipedia/Code/APIURLComponentsBuilder.swift

```diff
@@ -120,12 +137,109 @@ public struct APIURLComponentsBuilder {
             return APIURLComponentsBuilder(hostComponents: components, basePathComponents: Configuration.Path.mediaWikiRestAPIComponents)
         }
         
+        private static func productionRestBuilder(withLanguageCode languageCode: String? = nil) -> APIURLComponentsBuilder {
+            let host: String
+            if let languageCode = languageCode {
+                host = "\(languageCode).\(Configuration.Domain.wikipedia)"
+            } else {
+                host = Configuration.Domain.englishWikipedia
+            }
+            return productionRestBuilder(withWikiHost: host)
+        }
+        
         private static func productionBuilder(withWikiHost wikiHost: String? = nil) -> APIURLComponentsBuilder {
             var components = URLComponents()
             components.host = wikiHost ?? Configuration.Domain.metaWiki
             components.scheme = Configuration.Scheme.https
             return APIURLComponentsBuilder(hostComponents: components, basePathComponents: Configuration.Path.mediaWikiAPIComponents)
         }
+        
+        private static func productionBuilder(withLanguageCode languageCode: String? = nil) -> APIURLComponentsBuilder {
```

Comment:
Agh, great catch. These new language code builder methods were an original attempt to build beta cluster endpoints, but then I found a more simple way to make these with the original host builder methods by using the [defaultSiteDomain](https://github.com/wikimedia/wikipedia-ios/blob/048f97ade3464c01042326dcce5ee24292f13418/WMF%20Framework/Configuration.swift#L310) instead of hardcoding like it was before. I pushed a commit deleting these unused new methods.